### ISSUE TITLE
Unit tests for space_age exercise do not drive to a correct implementation

### ISSUE THREAD

**Issue by tronthomas** [exercism/problem-specifications#1626]
The tests written for the space_age exercise appear to be insufficient and do not seem to drive someone toward creating a correct implementation.

Following the TDD principle of doing the simplest thing to make a test pass, for all the tests, someone can simply hard code the expected value and everything will pass.

**Comment by yawpitch** [comment]
That’s true of more than just this exercise, there are several that could be “correctly” answered by just mapping each canonical input with its expected output.

Personally I feel that this is where the mentors come in; in a real world situation such a simplistic approach to TDD would simply get you fired, and rightfully so. If you submit such a solution to the mentor queue you’ll get a gentler form of pushback. If we diligently ensure that every solution cannot be solved we’re just encouraging people to believe that the tests describe _correct_ functionality instead of _minimal_ functionality, which is something TDD tests never do in the real world.

That said if you’d like to submit a PR with an extra test or two for each planet we could merge it after #1560 is resolved. You may find it difficult to design a set of tests for this exercise and the interface it describes that _cannot_ be passed by a sufficiently large lookup table, though.

**Comment by ErikSchierboom** [comment]
@tronthomas Are you still interested on working on this issue?

> That’s true of more than just this exercise, there are several that could be “correctly” answered by just mapping each canonical input with its expected output.

That's totally correct. 

> That said if you’d like to submit a PR with an extra test or two for each planet we could merge it after #1560 is resolved.

I think an extra test or two per planet could make sense.

**Comment by IsaacG** [comment]
Closing as abandoned.
